<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gaming</class>
 <widget class="QMainWindow" name="Gaming">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>动物连连看</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>900</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/jpg/source/游戏ing界面/5a2fdd9ee6a14_1024.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>62</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 170, 127);
background-color: rgb(170, 170, 127);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/new/jpg/source/提示/B/pauseB.png</normaloff>:/new/jpg/source/提示/B/pauseB.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="TimeLeft">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>15</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>790</y>
      <width>95</width>
      <height>95</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/new/jpg/source/提示/S/hintS.png</normaloff>:/new/jpg/source/提示/S/hintS.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>95</width>
      <height>95</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>790</y>
      <width>95</width>
      <height>95</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/new/jpg/source/提示/S/restartS.png</normaloff>:/new/jpg/source/提示/S/restartS.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>95</width>
      <height>95</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>790</y>
      <width>95</width>
      <height>95</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/new/jpg/source/提示/S/backS.png</normaloff>:/new/jpg/source/提示/S/backS.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>95</width>
      <height>95</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>742</x>
      <y>790</y>
      <width>95</width>
      <height>95</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/new/jpg/source/提示/S/musicS.png</normaloff>
      <normalon>:/new/jpg/source/提示/volume.png</normalon>
      <disabledon>:/new/jpg/source/提示/S/musicS.png</disabledon>
      <activeon>:/new/jpg/source/点击效果初步/4.jpg</activeon>:/new/jpg/source/提示/S/musicS.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>95</width>
      <height>95</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>90</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ink Free</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 157)</string>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>55</y>
      <width>710</width>
      <height>710</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>256</width>
      <height>128</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icon.qrc">:/new/jpg/source/游戏ing界面/score.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Branch">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>571</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Monkey">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>53</width>
      <height>44</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>TimeLeft</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_3</zorder>
   <zorder>Branch</zorder>
   <zorder>Monkey</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
